lib/src/rules/no_leading_underscores_for_local_identifiers.dart

```diff
@@ -65,6 +65,8 @@ class NoLeadingUnderscoresForLocalIdentifiers extends LintRule
     registry.addDeclaredIdentifier(this, visitor);
     registry.addFormalParameterList(this, visitor);
     registry.addForPartsWithDeclarations(this, visitor);
+    registry.addFunctionDeclaration(this, visitor);
```

Comment:
Could this registry entry be safely removed, as we are not using it?

Reply:
Oh dear.  Great catch.  Thanks!

https://github.com/dart-lang/linter/pull/3364